src/Entities/Minecart.cpp

```diff
@@ -514,7 +517,7 @@ void cMinecart::HandlePoweredRailPhysics(NIBBLETYPE a_RailMeta)
 			SetSpeedY(0);
 			SetSpeedX(0);
 
-			bool BlckCol = TestBlockCollision(a_RailMeta), EntCol = TestEntityCollision(a_RailMeta);
+			bool BlckCol = TestBlockCollision(PoweredRailMeta), EntCol = TestEntityCollision(PoweredRailMeta);
```

Comment:
Why use the metadata in the seventh bit? Please add a note about that for the next person touching the code. The old version is more intuitive? Or does the rail block only contain the seventh bit?

Reply:
PoweredRail only has 5 status in low 3bit. so we need do a logical and to get correct powered rail meta data.
The old version was confused me when I review this code ><